Source/WebKit/WebProcess/Extensions/API/WebExtensionAPIStorageArea.h

```diff
@@ -34,43 +34,40 @@
 
 namespace WebKit {
 
-enum class StorageType { Local, Session, Sync };
-
 class WebExtensionAPIStorageArea : public WebExtensionAPIObject, public JSWebExtensionWrappable {
     WEB_EXTENSION_DECLARE_JS_WRAPPER_CLASS(WebExtensionAPIStorageArea, storageArea);
 
 public:
 #if PLATFORM(COCOA)
     bool isPropertyAllowed(ASCIILiteral propertyName, WebPage*);
 
-    void get(id items, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
-    void getBytesInUse(id keys, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
-    void set(NSDictionary *items, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
-    void remove(id keys, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
-    void clear(Ref<WebExtensionCallbackHandler>&&);
-
-    // Exposed only by storage.session.
-    void setAccessLevel(NSDictionary *, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
-
-    WebExtensionAPIEvent& onChanged();
+    void get(WebPage *, id items, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
```

Comment:
Since `WebPage` is a C++ type, the `*` should be next to the type. There are also some instances in the implementation files in this patch that need to be corrected.

```suggestion
    void get(WebPage*, id items, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
```